tikv-configuration-file.md

```diff
@@ -369,8 +369,13 @@ Configuration items related to storage.
 
 ### `enable-ttl`
 
+> **Warning:**
+>
+> - The TTL feature is only available for the RawKV interface for now. You can only configure this feature when creating a new cluster because TTL uses different data formats in the storage layer.
```

Comment:
```suggestion
> - Currently, the TTL feature is available only for the RawKV interface. You can configure TTL **ONLY WHEN** creating a new cluster because TTL uses different data formats in the storage layer.
```